AllCoreDevs-Meetings/Meeting 146.md

```diff
@@ -0,0 +1,174 @@
+
+**Ansgar Dietrichs**
+* Yeah, I was just wondering if you, if you already have a rough idea of when people can expect that the release, 
+
+**Lukaz Rozmej**
+* So we will start a good early next week. We'll be probably on Tuesday. Sounds good, But if you're running one thing, zeros. 
+
+**Ansgar Dietrichs**
+* Okay. Quick context. Gary. 
+
+**Gary Schulte**
+* Yeah. similarly, basically Besu it was going to be doing a release next week on Wednesday, the seventh, 20.7.2, there's. We don't have a critical bug that we're trying to fix, but it's just, you know, logging improvements. some additional, features not necessarily features just, fixes, that will be, not critical, but recommended for, the merge as that I believe the seventh is post Bellatrix, but, it shouldn't be available on the seventh and we will be recommending that release for the merge. 
+
+**Ansgar Dietrichs**
+* Sounds good. Sounds like a lot of, visibly small, optional, intro leases. I mean, I think this is, that's a great place to be in just this smallest minute, improvements. Do we have any, any other, merge-related items for today? Okay. 
+
+**Mikhail Kalinin**
+* I mean, Yeah, I guess we have like, yeah, one another small one. So we do want to give an update on the optimistic seen tests. 
+
+**Hsiao-Wei Wang**
+* Ah, yes. So, we have a consensus, spec text factor up dates here. Let me put the link here in the chat. So I'm Kyle and I are working on the optimistic test vectors. And so we, I attached the, the simple test vector TAROS in this PR. So, we are looking for more inputs from the consensus layer, a client, developers on the new test vector, format, especially. And so in this goal that, it would be great if we have at this one or maybe better two plantings to, implement the basic test runner and provide some feedback, for us before we merged this PR and make, no more release, for the whole test suit. So yeah, PR here, a request for feedback thinking. 
+
+**Ansgar Dietrichs**
+* Okay. great. maybe just because, I see some comments in the chat, I think, what else is still predicting more or less the same, same time for the merge, Thursday at around 4:00 AM, UTC of course, a little bit of variance depending on the country, hash rates. So actually that would be just for context, if you ask before our next, all core dev call. So, I think there's a very decent chance that this one is the last step, pretty much, a all coredev call, but you, who knows if, if the, if the hash rate, ends up dropping a little bit before the merge, we might, we might, just barely, slipped to after, after the call. And also, I think there are a few other kind of, merge, prediction websites out there they'd have slightly different estimates instead. Do we any chance to know specificlly. I mean, of course there's some uncertainty, but, I think Battelle has been so far kind of the most precise. so is that kind of the main place for people to, to look to, to be, not anything there? Mo Maria? 
```

Comment:
```suggestion
* Okay. great. maybe just because, I see some comments in the chat, I think, what else is still predicting more or less the same, same time for the merge, Thursday at around 4:00 AM, UTC of course, a little bit of variance depending on the country, hash rates. So actually that would be just for context, if you ask before our next, all core dev call. So, I think there's a very decent chance that this one is the last step, pretty much, a all coredev call, but you, who knows if, if the, if the hash rate, ends up dropping a little bit before the merge, we might, we might, just barely, slipped to after  the call. And also, I think there are a few other kind of, merge, prediction websites out there they'd have slightly different estimates instead. Do we any chance to know specifically. I mean, of course there's some uncertainty, but, I think Bellatrix has been so far kind of the most precise. so is that kind of the main place for people to, to look to, to be, not anything there? Mo Maria? 
```